private-undistributed.yml

```diff
@@ -65,6 +65,7 @@ allow-licenses:
   - 'LGPL-3.0'
   - 'LGPL-3.0-only'
   - 'LGPL-3.0-or-later'
+  - 'LGPL-3.0-or-later WITH openvpn-openssl-exception'
```

Comment:
From what I see, openvpn-openssl-exception is not a valid license ID. Valid IDs are https://spdx.org/licenses/.

I'm not sure that the Action will work if we add this new expression here.

Reply:
Oh yeah, I see, it's an exemption rather than a license.

The docs for the action (https://github.com/actions/dependency-review-action) say `allow-licenses` can be "Any SPDX-compliant identifier(s)"

On the page you link, it says:
> License Exceptions are commonly found exceptions to free and open source licenses, used with the License Expression operator, "WITH" to create a license with an exception.

`openvpn-openssl-exception` in on the list here: https://spdx.github.io/license-list-data/exceptions-index.html

Reply:
Alright, I guess we can try it 😅 

Worst case we break everyone's CI and revert.

Reply:
This repo requires two approvals.

Reply:
I meaning merging my original PR, where the license check came up: https://github.com/coveo-platform/audit-dashboard/pull/51

Reply:
It looks like I can disregard the license warning on my PR. Would you rather I do that, or go ahead with this change?